velox/exec/HashProbe.cpp

```diff
@@ -481,11 +482,19 @@ RowVectorPtr HashProbe::getOutput() {
           ++numOut;
         }
       }
+    } else if (isAntiJoin(joinType_) && !filter_) {
```

Comment:
This branch can be merged with the previous one?

Reply:
Not easily, the null condition is inverted.

Reply:
```
I see thanks, and then remove the bracket inside the if condition?

!nonNullInputRows_.isValid(i) || !activeRows_.isValid(i) || !lookup_->hits[i]

Do you know in which case activeRows_.isValid(i) is true but lookup_->hits[i] is null?
```